src/open_inwoner/accounts/views/profile.py

```diff
@@ -149,6 +153,64 @@ def form_valid(self, form):
         return HttpResponseRedirect(self.get_success_url())
 
 
+class MyDataView(
+    LogMixin, LoginRequiredMixin, CommonPageMixin, BaseBreadcrumbMixin, TemplateView
+):
+    template_name = "pages/profile/mydata.html"
+
+    @cached_property
+    def crumbs(self):
+        return [
+            (_("Mijn profiel"), reverse("accounts:my_profile")),
+            (_("Mijn gegevens"), reverse("accounts:my_data")),
+        ]
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context["my_data"] = self.parse_brp_data()
+        return context
+
+    def parse_brp_data(self):
+        brp_version = settings.BRP_VERSION
+        user = self.request.user
+        my_data = {
+            "first_name": "naam.voornamen",
+            "initials": "naam.voorletters",
+            "last_name": "naam.geslachtsnaam",
+            "prefix": "naam.voorvoegsel",
+            "birthday": "geboorte.datum.datum",
+            "birthday_place": "geboorte.plaats.omschrijving",
+            "birthday_country": "geboorte.land.omschrijving",
+            "gender": "geslachtsaanduiding.omschrijving"
+            if brp_version == "2.0"
+            else "geslachtsaanduiding",
+            "street": "verblijfplaats.straat",
+            "house_number": "verblijfplaats.huisnummer",
+            "postcode": "verblijfplaats.postcode",
+            "place": "verblijfplaats.woonplaats",
+            "land": "verblijfplaats.land.omschrijving",
+        }
+
+        self.log_user_action(user, _("user requests for brp data"))
+
+        fetched_data = fetch_brp_data(user, brp_version)
+
+        # we have a different response depending on brp version
+        if brp_version == "2.0" and fetched_data.get("personen"):
+            fetched_data = glom(fetched_data, "personen")[0]
```

Comment:
You don't need to glom if it is only one level deep `fetched_data["personen"][0]`